scripts/PyChop/ISISDisk.py

```diff
@@ -338,6 +337,7 @@ def plotFrame(self, h_plt=None, Ei_in=None, frequency=None):
         Plots the time-distance diagram into a given Matplotlib axes, i
         for a give focused incident energy (in meV) and chopper frequencies (in Hz).
         """
+        from matplotlib import pyplot
```

Comment:
You need to import this only if `h_plt==None`. And I would use  `try...except ImportError`